README.md

```diff
@@ -1,33 +1,36 @@
-# Operate First template for repositories
+# Community Handbook
 
-Derive new repositories from this template
+## Welcome to the Community Handbook for Operate First!
 
-List of featurese:
+The Community Handbook is intended to serve as a resource for all Operate First community contributors.
+It contains documentation on how to perform contributor-related tasks and encourages input and contributions from the community.
+The covenant of the Community Handbook is that if contributors must figure out a task for themselves, they should submit that solution to the handbook for the benefit of future contributors.
 
-## License
+## How to Contribute
 
-This template ensures new repos are created compliant with [ADR 0001](https://www.operate-first.cloud/blueprints/blueprint/docs/adr/0001-use-gpl3-as-license.md) and use GNU GPL v3 license.
+We welcome the Operate First community to contribute to the handbook!
+There are multiple ways to contribute:
+1. Create an issue in our [GitHub repository](https://github.com/operate-first/community-handbook)
+1. Create a pull request (PR) directly against the repository
+1. Label an issue or PR from another part of the project to identify as part of the Community Handbook
```

Comment:
should the issue be labeled /area community, if so should we specify it here?

Reply:
@quaid and I discussed which labels to apply and chose area/contributor and kind/handbook to signify issues and prs for the Community Handbook. In the future, one of us will write out explicit directions for how to link other parts of the project to the handbook, but we just haven't gotten to it yet!